### PR TITLE
Feature (userprofile)Users should be able to view and edit their prof…

### PR DESCRIPTION
**Pull request by mulondo**
**What does this PR do?**
This Pull Request adds the following functionality to the articles app:
	•user can view his or her profile

**Description of Task to be completed?**
The logged in user can view and edit his or her profile information which includes the photo, the bio and the fun fact. 

**How should this be manually tested?**
- Install dependencies: `yarn`
- Run the application server: `yarn start`
- To see the profile go to `https.zeus-fe.herokuapp.com/profile` in the browser

**What are the relevant pivotal tracker stories?**
[#162163133](https://www.pivotaltracker.com/story/show/162163133)

**Screen shots
- user profile
screen shot 2019-01-29 at 14 15 19

- Edit userprofile
screen shot 2019-01-29 at 14 14 09

### CODE REVIEW

**Review by sekayasin — APPROVED** [review]
Bravo, LGTM

**Review by alexkayabula — APPROVED** [review]
Looks good

File: src/__tests__/reducers/userProfile/userprofile.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+let state ={};
+// const initialStore = {
+//   userprofile: {}
+// };
+
+// describe('test auth reducer', () => {
+//   it('userprofile reducer', () => {
+//     const newUser = { userprofile: {}, loading: false };
+//     expect(
+//       userprofile(undefined, { type: USER_PROFILE, payload: newUser })
```
Comment (barema4) [inline src/__tests__/reducers/userProfile/userprofile.test.js]: Please remove commented code

File: src/__tests__/reducers/userProfile/userprofile.test.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  const action = {
+    type: USER_PROFILE,
+    payload: {
+      fun_fact:"swimming",
+      bio: "I love android",
+      photo:
+        "https://res.cloudinary.com/dksxmwjqs/image/upload/v1544192758/hqqg0sil5sifcvigi5lx.jpg",
+    }
+  };
+  console.log(">>>>>>>>>>>>>>>>>>>>",userprofile);
```
Comment (barema4) [inline src/__tests__/reducers/userProfile/userprofile.test.js:21]: Kindly remove the console.log

**Review by alexkayabula — COMMENTED** [review]
Remove commented out code.

File: src/__tests__/reducers/userProfile/userprofile.test.js (hunk 1 above)
Comment (mulondo) [reply src/__tests__/reducers/userProfile/userprofile.test.js]: that is sorted

**Pull request merged**